DependencyInjection/Configuration.php

```diff
@@ -34,6 +34,17 @@ public function getConfigTreeBuilder()
                     ->prototype('scalar')
                     ->end()
                 ->end()
+                ->arrayNode('frontend')
+                  ->prototype('array')
+                    ->children()
+                      ->arrayNode('extensions')
+                        ->prototype('scalar')
+                        ->end()
+                      ->end()
+                      ->scalarNode('sequence')->end()
```

Comment:
Forcing our users to configure the regex themselves looks like a bad DX to me. Maintaining this regex for the common cases should be the bundle responsibility (you can make it configurable in the extractor class to make it generic, but then, please provide pre-configured services)

Reply:
Done. Hopefully the default configuration is alright.